static/app/views/alerts/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -105,7 +105,7 @@ const getFullActionTitle = ({
   'type' | 'integrationName' | 'sentryAppName' | 'status'
 >) => {
   if (sentryAppName) {
-    if (status) {
+    if (status && status !== 'published') {
       return `${sentryAppName} (${status})`;
```

Comment:
minor thing, but this might look nice as a `<Tag>` component.

Reply:
I tried it and I hate it lol, even if they were all like that it looks pretty weird to me.

<img width="560" alt="Screen Shot 2022-02-15 at 10 03 40 AM" src="https://user-images.githubusercontent.com/29959063/154121935-15d445b2-d9a8-4f06-b25e-f0dfd15c6f8f.png">